.travis.yml

```diff
@@ -29,5 +30,6 @@ matrix:
       language: generic
 
 script:
-  -  cd conda-recipes/gym-feedstock
+  -  cd conda-recipes/mxnet-feedstock
```

Comment:
I notice above you are using `CONFIG=linux_noarch` but this is not a noarch recipe. Change the CONFIG so it has one entry for each python version and drop the noarch references. Then you can also drop the `.ci_support/linux_noarch.yaml` file as well.

like:
```
matrix:
   include:
   -  env: CONFIG=linux_ppc64le_python3.6.yaml UPLOAD_PACKAGES=True DOCKER_IMAGE=condaforge/linux-anvil-ppc64le
      os: linux-ppc64le
      language:  generic

   -  env: CONFIG=linux_ppc64le_python3.7.yaml UPLOAD_PACKAGES=True DOCKER_IMAGE=condaforge/linux-anvil-ppc64le
      os: linux-ppc64le
      language:  generic
```

That way it will kick off jobs for the py36 and py37 builds. If this recipe works on x86, we'd like that enabled as well:

```
 -  env: CONFIG=linux_python3.6.yaml UPLOAD_PACKAGES=True DOCKER_IMAGE=condaforge/linux-anvil-comp7
      os: linux-64
      language:  generic

   -  env: CONFIG=linux_python3.7.yaml UPLOAD_PACKAGES=True DOCKER_IMAGE=condaforge/linux-anvil-comp7
      os: linux-64
      language:  generic
```